dependencies.gradle

```diff
@@ -0,0 +1,12 @@
+/*
+ * Build dependencies
+ */
+ext.androidCompileSdkVersion = 26
+ext.androidMinSdkVersion = 19
+
+ext.androidBuildToolsVersion = '26.0.2'
+
+ext.daggerVersion = "2.14.1"
+
+ext.kotlinVersion = '1.2.21'
+ext.gradleAndroidPluginVersion = '3.0.1'
```

Comment:
Not a *big* fan of breaking these out like this, but I'm probably gonna end up changing it a bit soon. So this is fine for now.

Reply:
I pulled out the dagger version because changing it is too much work for an update that would necessarily change the versions of all the modules in this project